From 7a2e6ada142803f35664aa17ac408459b522e12d Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Tue, 14 Jan 2020 09:26:01 +0200
Subject: [PATCH] [SW-7984] hl: update preboot offset in log

Offset is not start of Image (BTL) but HL_SPI_PREBOOT_OFF bytes away

Change-Id: I0a29abf9fbe2019f180497629fcbc8c1631d6d7d
---

diff --git a/app/ppboot.c b/app/ppboot.c
index 50f5d3e..cc67e48 100644
--- a/app/ppboot.c
+++ b/app/ppboot.c
@@ -47,7 +47,7 @@
 	u32 preboot_offset = img_offset + HL_SPI_PREBOOT_OFF;
 
 	preboot_puts_raw("Preparing to run Preboot stable");
-	preboot_puts_hex_raw(" (offset: ", img_offset, false);
+	preboot_puts_hex_raw(" (offset: ", preboot_offset, false);
 	preboot_puts(")\n");
 
 	spi_copy((void *)PREBOOT_TEXT_BASE, preboot_offset, PREBOOT_SIZE, name);
